test/fixtures/scss/export-option.scss

```diff
@@ -0,0 +1,15 @@
+[id="foo"] {
+    background: #f00;
+}
+
+[id^="bar"] {
+    background: #0f0;
+}
+
```

Comment:
I think we can assert the same functionality with just the `.foo` and `:export` rules or maybe just `:export` by itself

```suggestion
```